execution/executor/bond_test.go

```diff
@@ -80,6 +79,14 @@ func TestExecuteBondTx(t *testing.T) {
 		assert.Equal(t, errors.Code(err), errors.ErrInvalidPublicKey)
 	})
 
+	t.Run("Should fail, amount less than MinimumStake", func(t *testing.T) {
+		trx := tx.NewBondTx(td.stamp500000, senderAcc.Sequence()+1, senderAddr,
```

Comment:
I think we changed `stamp500000` to `randStamp`.

@kehiy run `make unit_test` locally.

Reply:
the errors is from state test